articles/virtual-machines/flash-azure-monitor.md

```diff
@@ -43,6 +43,17 @@ To navigate to the [metrics explorer](/azure/azure-monitor/essentials/metrics-ge
 
 For a description of the metric's display values, see [VM availability metric](monitor-vm-reference.md#vm-availability-metric-preview).
 
+Users can split the VM availability by the 'Context' property.
+   :::image type="content" source="media/flash/context-property-filter.png" alt-text="Screenshot of the newly added Context property of VM availability Metric on Azure portal." lightbox="media/flash/context-property-filter.png" :::
+
+Note: The context for migrations (Service Healing and Live Migration) is unknown by default. We are working on making this information available in the future.
```

Comment:
```suggestion
> [!NOTE]
> The context for Service Healing and Live Migration is currently unknown by default.
```

Reply:
We are not allowed to make promises about the future/products or things coming up. Please completely remove that language.